### PR TITLE
CDS-18: checked list

### PR DESCRIPTION
**Pull request by liljosi**

### CODE REVIEW

**Comment by diegoaguilar** [comment]
@liljosi you didn't add the fonts folder yet

File: src/components/header/header.js [hunk 1]
```diff
@@ -26,7 +26,7 @@ const Header = ({ title }) => (
   <AnimatedContainer>
     <Container>
       <Link to="/">
-        <Title as="h1">{title}</Title>
+        <Title as="h1">{}</Title>
```
Comment (diegoaguilar) [inline src/components/header/header.js:29]: This works but would become `AnimatedContainer` not reusable.

I think it's more like making the "white" section not to use `header.js` or just turning it into another components

**Comment by liljosi** [comment]
That’s not the the branch for fonts, that would be CDS-4

File: src/components/intro/intro.css [hunk 2]
```diff
@@ -21,4 +21,29 @@
...
+  }
+  .introContainer {
+    height: 100vh;
+    width: 100%;
+    background-color: #f0f0e9;
+    display: flex;
+    flex-direction: row;
+    justify-content: center;
+    align-items: center;
+    background-image: radial-gradient(
```
Comment (diegoaguilar) [inline src/components/intro/intro.css]: Can you explain me what this is doing. I'm not saying that is wrong, just that I'd like to understand its function.

File: src/components/intro/intro.css [hunk 3]
```diff
@@ -21,4 +21,29 @@
...
+}
+
+@media only screen and (max-width: 768px) {
+  /* For mobile phones: */
+  .logoContainer {
+    height: 45vw;
+    width: 45vw;
+    margin-right: 0%;
+  }
+  .introContainer {
```
Comment (diegoaguilar) [inline src/components/intro/intro.css]: Use the `BEM` naming convention that we just discussed .... **Everywhere**

**Comment by diegoaguilar** [comment]
Fix the actionable items from my comments, then take care of the opaque background on the logo so the C is not mixed up with the dots pattern.

**Pull request merged**